tests/e2e/lib.sh

```diff
@@ -237,6 +237,8 @@ deploy_central_via_operator() {
     ROX_PASSWORD="$(tr -dc _A-Z-a-z-0-9 < /dev/urandom | head -c12 || true)"
     centralAdminPasswordBase64="$(echo "$ROX_PASSWORD" | base64)"
 
+    centralAdditionalCA="$(sed 's,^,        ,' "${TRUSTED_CA_FILE}")"
```

Comment:
Why do we need to indent every line? Can you add a comment with explanation or add it in PR description?

Reply:
Renamed the variable to make it self-explanatory.